doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md

```diff
@@ -89,9 +89,7 @@ And we also need two corresponding test classes:
    }
    ```
   
-6. **Add the dialect name** (here `ATHENA`) **to the list of dialects**  for which the `JbdcAdapterFactory` is responsible in the method `getSupportedAdapterNames()`.
-
-7. Create a **new unit test class** for the dialect in the test package you created before: `com.exasol.adapter.dialects.athena.AthenaSqlDialectTest` that tests class `AthenaSqlDialect`.
+6. Create a **new unit test class** for the dialect in the test package you created before: `com.exasol.adapter.dialects.athena.AthenaSqlDialectTest` that tests class `AthenaSqlDialect`.
```

Comment:
Use `1.` for numbers to make your life easier.

Reply:
fixed